jetty-io/src/main/java/org/eclipse/jetty/io/ByteBufferPool.java

```diff
@@ -78,6 +78,11 @@ default ByteBuffer newByteBuffer(int capacity, boolean direct)
         return direct ? BufferUtil.allocateDirect(capacity) : BufferUtil.allocate(capacity);
     }
 
+    default RetainableByteBufferPool asRetainableByteBufferPool()
+    {
+        return RetainableByteBufferPool.from(this);
+    }
+
     public static class Lease
```

Comment:
@sbordet is `Lease` really the right name for this?  It's kind of a `ByteBufferCollection`?    Eitherway, can you add some javadoc for the class